README.md

```diff
@@ -59,7 +77,7 @@ or
 <script type="text/javascript" src="lozad.min.js"></script>
 ```
 
-**Note**: `lozad.js` supports AMD and commonJS module pattern out of the box.
+You can find the library on `window.lozad`.
```

Comment:
already mentioned on line 67, why to repeat?

Reply:
Done.